### DISCUSSION TITLE
Add a git section to the FAQ

### DISCUSSION THREAD

**Discussion by AureliaDolo** [massalabs/massa#1964] [Documentation]
Especially with the transition from gitlab to github 

What do you think @Noxhaa

**Comment by Noxhaa** [comment]
Could be useful, but I don't get specific questions about it, so it's gonna be a bit general on how git works

**Comment by AureliaDolo** [comment]
Maybe when someone has an issue with git you can get into the habit of writing the answer in the doc. I've seen someone with a fatal: destination path 'massa' already exists and is not an empty directory.  git error, maybe it was the only one x)

**Comment by Noxhaa** [comment]
I'll lookout and note massa specific git issues :)

**Comment by AureliaDolo** [comment]
I move that to discussion. Feel free to edit the wiki when you encounter recurring issues :)
